.github/workflows/run-and-test-builds.yml

```diff
@@ -37,8 +37,6 @@ jobs:
         with:
           node-version: lts/*
           cache: 'yarn'
-      - name: Ignore monorepo postcss.config.js
-        run: mv postcss.config.js _postcss.config.js
```

Comment:
I thought this was in there because without it the build failed. @wlee221 mentioned it. Maybe it's no longer an issue?

Reply:
Right, this will not be an issue after we move that monorepo postcss config.

Reply:
The problem was that canaries would find this `postcss.config.js` in the monorepo root and try to run postcss. Removing `postcss.config.js` will prevent this from happening at all.

Reply:
With the `postcss.config.js` file gone, it's no longer an issue